deepdrivemd/sim/openmm_stream/run_openmm.py

```diff
@@ -0,0 +1,207 @@
+import shutil
+import simtk.unit as u
+import simtk.openmm as omm
+from mdtools.openmm.sim import configure_simulation
+from deepdrivemd.utils import Timer, parse_args
+from deepdrivemd.sim.openmm.config import OpenMMConfig
```

Comment:
Importing the wrong config class, should be `from deepdrivemd.sim.openmm_stream.config`. Silent errors due to having two separate data classes (one in openmm_stream/config.py, and the other in the YAML generation script).

Reply:
Fixed